ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -5,27 +5,21 @@ package com.daml.error.definitions
 
 import com.daml.error._
 import com.daml.error.definitions.ErrorGroups.ParticipantErrorGroup.LedgerApiErrorGroup
-import com.daml.ledger.participant.state.v2.ChangeId
-import com.daml.lf.data.Ref
-import com.daml.lf.data.Ref.PackageId
 import com.daml.lf.engine.Error.Validation.ReplayMismatch
 import com.daml.lf.engine.{Error => LfError}
-import com.daml.lf.interpretation.{Error => LfInterpretationError}
-import com.daml.lf.language.{LanguageVersion, LookupError, Reference}
-import com.daml.lf.transaction.GlobalKey
-import com.daml.lf.value.Value
-import com.daml.lf.{VersionRange, language}
 import org.slf4j.event.Level
-import java.time.{Duration, Instant}
-
-import scala.concurrent.duration._
 
 @Explanation(
   "Errors raised by or forwarded by the Ledger API."
 )
 object LedgerApiErrors extends LedgerApiErrorGroup {
 
   val Admin: groups.AdminServices.type = groups.AdminServices
+  val CommandExecution: groups.CommandExecution.type = groups.CommandExecution
+  val AuthorizationChecks: groups.AuthorizationChecks.type = groups.AuthorizationChecks
+  val ConsistencyErrors: groups.ConsistencyErrors.type = groups.ConsistencyErrors
+  val RequestValidation: groups.RequestValidation.type = groups.RequestValidation
+  val WriteServiceRejections: groups.WriteServiceRejections.type = groups.WriteServiceRejections
```

Comment:
Making the change source compatible, i.e. `LedgerApiErrors.CommandExecution...` still works